### PR TITLE
API v2: documentation

### PR DESCRIPTION
**Pull request by rowanseymour**
Adds API token to the formax summary so it's more obvious that this section relates to the API:

screen shot 2016-10-18 at 10 17 31

And reworks the form a little to separate out the API stuff from the webhook stuff:

screen shot 2016-10-18 at 10 20 02

screen shot 2016-10-18 at 10 17 50

And also fills API root page with documentation:

screen shot 2016-10-18 at 10 35 52

### CODE REVIEW

File: temba/api/v2/views.py [hunk 1]
```diff
@@ -38,14 +37,13 @@
 from ..support import InvalidQueryError
 
 
-@api_view(['GET'])
-@permission_classes((SSLPermission, IsAuthenticated))
-def api(request, format=None):
+class RootView(views.APIView):
```
Comment (rowanseymour) [inline temba/api/v2/views.py]: Might as well have this be a class-based view like everything else

**Comment by nicpottier** [comment]
Let's just remove the "you have no webhook configured" sentence, since we don't love those webhooks anyways

**Comment by rowanseymour** [comment]
Sure, tho do we continue to display the url if they do have one configured?

**Comment by nicpottier** [comment]
No, I'd say in both cases just show the API key.

**Comment by rowanseymour** [comment]
Updated:

screen shot 2016-10-18 at 20 19 46

**Pull request merged**